nodejs/week1/warmup.js

```diff
@@ -0,0 +1,20 @@
+class circle {
+  constructor(radius) {
+    this.radius = radius;
+  }
+
+  getArea() {
+    return Math.PI * this.radius * this.radius;
```

Comment:
instead of  `this.radius * this.radius` you can write Math.pow(this.radius, 2)
Makes code less repetitive.
You can read about this function here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow